src/wp-includes/class-wp-admin-bar.php

```diff
@@ -434,11 +434,7 @@ public function render() {
 		// We have to do this here since admin bar shows on the front end.
 		$class = 'nojq nojs';
 		if ( $is_IE ) {
```

Comment:
Nit: `if $is_IE && MSIE 9`

As this has been discussed in the Core-CSS channel, I've been going back and forth as to whether these classes should be removed or maintained due to back-compat. I've been leaning toward removal but figured best to raise it.

Reply:
Changed in https://github.com/WordPress/wordpress-develop/pull/251/commits/7c155d6ec783bca201c8523f3be86573bb09e51a. Although I think we can actually remove that since core has no specific .ie9 styles…

I think removing the classes fine as nothing should be fundamentally broken without them which isn't already broken due to IE being IE…